### DISCUSSION TITLE
Thoughts on implementing `Responder` for `()`?

### DISCUSSION THREAD

**Discussion by Aurora2500** [actix/actix-web#2669] [Ideas] [answered]
I think that it would make sense to implement `Responder` for `()`, such that it would simply reply with an empty 200 OK response.

Any thoughts on this?

**Comment by robjtede** [comment, marked as answer]
It was removed a while ago and isn't likely to come back due to it being a foot gun. Relevant context here: https://github.com/actix/actix-web/issues/1108

**Reply by Aurora2500** [reply]
I see. Seems like encouraging people to write out the specific `Responder` a function returns would fix that instead of telling Rust that it returns `impl Responder`. That way the Rust compiler can check it for you that you're returning the `Responder` you want. Thanks for linking me the previous discussions on it!
